src/renderer/ColorManager.ts

```diff
@@ -148,6 +140,15 @@ export class ColorManager implements IColorManager {
     this.colors.ansi[15] = this._parseColor(theme.brightWhite, DEFAULT_ANSI_COLORS[15]);
   }
 
+  public applyWebglSelectionColorHack() {
+    // HACK: while webgl renderer adds support for selection colors
```

Comment:
Let's move this into WebGL renderer where it's called and just assign the { css, rgba } object manually, that way the hacks are self contained in the webgl part 😃